WikiWatershed/model-my-watershed
# Add Enticing Account Creation Modal On Save

## Overview

The revamp tries to ease new users into the MMW experience. Clicking save on a project introduces them to creating an account via some enticing, as-yet-unwritten descriptive modal text. From there, they can choose to create an account or login.

**Wireframe**
![screen shot 2017-06-14 at 3 13 11 pm](https://user-images.githubusercontent.com/7633670/27153523-cb0f04fe-511f-11e7-9592-646f774d019a.png)

Connects #1861 
Connects #1862 

### Demo

![screen shot 2017-06-14 at 4 12 49 pm](https://user-images.githubusercontent.com/7633670/27153529-ce2508d2-511f-11e7-8dd0-d7791b1ba980.png)

### Notes

This branch is based off of #1966. The relevant commits start at 4ceb91c

## Testing Instructions

 * Pull, `./scripts/bundle.sh`
 * While not logged in, try to save a project...test out all three paths `[ No thanks | Login | Create an account ]`
 * Confirm there's no login modal on app start
